data/tooltips.txt

```diff
@@ -496,3 +496,6 @@ tip "no fuel?"
 
 tip "insufficient energy to fire?"
 	`This ship has insufficient energy storage to fire an installed weapon.`
+
+tip "negative attribute!"
```

Comment:
Negative attributes are not necessarily a bug.

Reply:
I thought it would be a bug because of the error message when trying to install such an outfit:

https://github.com/endless-sky/endless-sky/blob/de6b31469601aa8ef5d1ad356ebbd65416bbe1aa/source/OutfitterPanel.cpp#L472-L474